src/components/input-field/input-field.tsx

```diff
@@ -167,6 +173,9 @@ export class InputField {
                 {this.renderTrailingIcon()}
                 <div class="mdc-line-ripple" />
             </label>,
+            <div class="mdc-text-field-helper-line">
+                <div class="mdc-text-field-helper-text">{this.helperText}</div>
+            </div>,
```

Comment:
I haven't tried it yet, so maybe MDC takes care of it, but I suspect this should only be output if there is actually any text in `this.helperText`?

Reply:
⚡️